black_it/MCS.R

```diff
@@ -0,0 +1,105 @@
+# Reading the data
+
+mData <- read.csv("Data.csv")
```

Comment:
What is the content of the mData matrix? Specifically, I see it has 4 columns but what do those columns represent exactly?